package/kernel/cryptodev-linux/Makefile

```diff
@@ -10,12 +10,12 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=cryptodev-linux
-PKG_VERSION:=1.12
+PKG_VERSION:=1.13
 PKG_RELEASE:=1
```

Comment:
Please create a separate PR for the cryptodev-linux changes then we can already merge them.

Reply:
OK. I'll create another PR.
The version 1.12 just can't be built with kernel 6.6 and that's why I add this patch here.

Reply:
I've created PR: https://github.com/openwrt/openwrt/pull/14797
This commit will be remove once the PR is merged.